docs/getting-started.md

```diff
@@ -106,6 +106,18 @@ The result file will contain information about the `mime-types` package itself,
 information about each dependency. The scope names come from the package managers, for NPM packages these are usually
 `dependencies` and `devDependencies`, for Maven package it would be `compile`, `runtime`, `test`, and so on.
 
+By default, `analyzer-result.yml` will not contain any information, about scopes that can be excluded in later steps (for example, to skip downloading and scanning `devDependencies`). To specify, which scopes should be skipped, add `.ort.yml` configuration file to the root directory of the repo. For more details on config files see [Configuration File](config-file-ort-yml.md).
+
+In our case, we will add `[mime-types-path]/npm.ort.yml` with following content:
```

Comment:
This should be `[mime-types-path]/.ort.yml`.

Reply:
fixed